### DISCUSSION TITLE
Could not pass the verification

### DISCUSSION THREAD

**Discussion by WordlessEcho** [community/community#51443] [GitHub Education]
I have tried some proof of my Student identity. Including: proof from CSSD that issued by China goverment and student ID with official seal. Still could not pass the verification after 37 tries.

**Comment by samus-aran** [comment]
Hi @WordlessEcho  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by WordlessEcho** [comment]
It just said:
Rejected Submitted Mar 30, 2023
Unfortunately, we weren't able to approve your educational discount request for [@WordlessEcho](https://github.com/WordlessEcho). 

What happened?
Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.

**Comment by WordlessEcho** [comment]
Hi. I haven’t been graduating yet but my GitHub Education is already expired. Any help for me?

**Reply by samus-aran** [reply]
Hi @WordlessEcho 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. **Student ID**) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
